circle.yml

```diff
@@ -115,5 +157,68 @@ workflows:
   version: 2
   build-test-and-deploy:
     jobs:
-      - build
+      - build:
+          name: quality_migrations_tests
+          tests_group: 0
+          tests_type: clean test.quality test.migrations_missing
+      - build:
+          name: python_unit_tests
+          tests_group: 0
+          tests_type: clean test.unit
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: browser_tests
+          tests_group: 0
+          tests_type: clean test.browser
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: js_tests
+          tests_group: 0
+          tests_type: clean test.js
+          requires:
+            - quality_migrations_tests
+      - build:
+          name: integration_tests_group_1
+          tests_group: 1
+          tests_type: test.integration
+          requires:
+            - python_unit_tests
+            - browser_tests
+            - js_tests
+      - build:
+          name: integration_tests_group_2
+          tests_group: 2
+          tests_type: test.integration
+          requires:
+            - python_unit_tests
+            - browser_tests
+            - js_tests
+      - build:
+          name: integration_tests_group_3
+          tests_group: 3
+          tests_type: test.integration
+          requires:
+            - python_unit_tests
+            - browser_tests
+            - js_tests
+      - coverage:
+          name: coverage
+          requires:
+            - integration_tests_group_1
+            - integration_tests_group_2
+            - integration_tests_group_3
+              #- python_unit_tests
+              #- browser_tests
+              #- js_tests
+  cleanup-workflow:
+    triggers:
+      - schedule:
+          cron: "0 1 * * *"
```

Comment:
Do we get any sort of logs for this cron job?

Reply:
@viadanna It will run as a regular job so it'll show up in the job list. It'll get [scheduled](https://circleci.com/docs/2.0/workflows/#scheduling-a-workflow) when there's a push to master because of the filter.

Reply:
:+1: